app/src/test/java/ch/epfl/sweng/swenggolf/FakeDatabaseTest.java

```diff
@@ -264,5 +284,24 @@ public void onCancelled(DbError error) {
         };
     }
 
+    class ValueTestListener<T> implements ValueListener<T> {
+        public int calls = 0;
+
+        @Override
+        public void onDataChange(T value) {
+            if (calls == 0) {
+                assertThat(value, (Matcher<? super T>) is(CONTENT));
```

Comment:
That cast is weird, is it really needed?

Reply:
No, you're right. I changed it, it should be fixed by now.